recipes/miniaudio/all/test_package/CMakeLists.txt

```diff
@@ -5,3 +5,13 @@ find_package(miniaudio REQUIRED)
 
 add_executable(test_package example.c)
 target_link_libraries(test_package miniaudio::miniaudio)
+
+if(UNIX AND NOT APPLE)
+  set(LINUX TRUE)
+endif()
+
+if(LINUX)
+  target_link_libraries(test_package m pthread dl)
+elseif(UNIX)
+  target_link_libraries(test_package m pthread)
+endif()
```

Comment:
Please, use `self.cpp_info.system_libs.append` in the conan recipe itself.
This way it will magically work for all the consumers of the library.

Reply:
Thanks. You and @SpaceIm noticed it at the same time. Sorry, new at making recipes :D